### PR TITLE
Changelog for 1.36.0

### PR DESCRIPTION
**Pull request by rgrunber**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 1.36.0 (October 30th, 2024)
+ * enhancement - Support using `javac` as a compiler. See [#3558](https://github.com/redhat-developer/vscode-java/pull/3558), [JLS#3167](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/3167).
+ * enhancement - Automatically add `///` on new line when editing Markdown javadoc. See [#3801](https://github.com/redhat-developer/vscode-java/issues/3801).
```
Comment (rgrunber) [inline CHANGELOG.md:5]: maybe say "Markdown comments"

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## 1.36.0 (October 30th, 2024)
+ * enhancement - Support using `javac` as a compiler. See [#3558](https://github.com/redhat-developer/vscode-java/pull/3558), [JLS#3167](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/3167).
```
Comment (rgrunber) [inline CHANGELOG.md:4]: Add more detail on how to enable this and the DOM completion engine.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,23 @@
 # Change Log
 
+## 1.36.0 (October 31st, 2024)
+ * enhancement - Experimental support for using `javac` as the compiler. See [#3558](https://github.com/redhat-developer/vscode-java/pull/3558), [JLS#3167](https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/3167).
+   * `java.jdt.ls.javac.enabled`: Enables Javac-based compilation.
+     * Requires running with **Java 23**.
```
Comment (testforstephen) [inline CHANGELOG.md:6]: -> Requires running with **Java 23**. Make sure `java.jdt.ls.java.home` points to `/path/to/jdk23/home`.

Comment (rgrunber) [reply CHANGELOG.md:6]: Good idea. Was considering whether we should explicitly mention that setting.

Comment (rgrunber) [reply CHANGELOG.md]: I went with "Make sure to use `java.jdt.ls.java.home` for this" . Let me know if that works for you.

Comment (testforstephen) [reply CHANGELOG.md]: yes, the new message is fine for me.

**Review by testforstephen — APPROVED** [review]

**Pull request merged**